name-emu/.vscode/launch.json

```diff
@@ -19,7 +19,7 @@
                     "kind": "bin"
                 }
             },
-            "args": ["63321"],
+            "args": ["63321", "/home/qwe/Documents/CS4485/name/mips_test.asm", "/home/qwe/Documents/CS4485/name/name-as/output.o"],
```

Comment:
Should this launch.json be in the repo?

Reply:
mmm, maybe. It shows an example of how to run this thing. Do you have a better suggestion?

Reply:
Maybe just documented in a README or something, I don't like the hardcoded paths